### ISSUE TITLE
Using the mutli issue file requires reduntant information

### ISSUE THREAD

**Issue by black0range** [repobee/repobee-feedback#4]
Using the issue file requires specifying the master repo name and if all students aren't in the file it will simply complain about missing students and crash. 

It should be changed to simply go with whatever is in the file, and simply warn that students are missing.

**Comment by slarse** [comment]
You could, but that would change the semantics of the program. Almost all of RepoBee's commands operate on the cross product of master repo names and student usernames, it's at the very core of the whole system. The complaint on missing elements of that cross product is a safety precaution. It actually saved me from missing to report a student's grades just last week because of a typo, so I will not remove it. Right now, the multi issue file is just a separate source format for issues, I don't want it to change the semantics of the rest of the plugin's execution. That would really call for another plugin.

There's also the implementation detail that the `--master-repo-names` and `--students` options are not actually part of this plugin, they are added by RepoBee's core parser.

**To summarize**, we could add e.g. `--warn-on-missing` flag that simply turns the crash on missing issues into a warning, but the master repo names and students will still need to be supplied.

**Comment by slarse** [comment]
@black0range So, would you like the `--warn-on-missing` flag? Not gonna do it if it does not at least somewhat solve your problem.

**Comment by black0range** [comment]
Well yes, it sounds like a good plan, however it also increases the complexity of the tool such that other people may find it confusing. I am unsure about what to do. Maybe it is best to have a `--allow-missing` flag instead?

**Comment by slarse** [comment]
Yeah, could do `--allow-missing` instead. I don't think it's a particularly complicated addition.
